tests/power_cycle/test_powercycletools.py

```diff
@@ -0,0 +1,96 @@
+from pprint import pprint
+
+import matplotlib.pyplot as plt
+import pytest
+
+from bluemira.base.look_and_feel import bluemira_print
+from bluemira.power_cycle.tools import (
+    _add_dict_entries,
+    _join_delimited_values,
+    adjust_2d_graph_ranges,
+    validate_axes,
+)
+
+# Simple vectors for sample plotting
+sample_x = [1, 2]
+sample_y = [3, 4]
+
+bluemira_print("Test Power Cycle Tools")
+
+
+def test_add_dict_entries(visualize=None):
+    old_dict = {
+        "value_1": 1,
+        "value_2": 2,
+        "value_3": 3,
+    }
+    new_entries = {
+        "value_1": 1,
+        "value_4": 4,
+    }
+    new_dict = _add_dict_entries(old_dict, new_entries)
+    if visualize:
+        bluemira_print("Test Power Cycle Tools ('_add_dict_entries')")
+        pprint(old_dict)
+        pprint(new_entries)
+        pprint(new_dict)
```

Comment:
> otherwise it will not be in our logging system.

Ah, I did not know that. I have added any `print` call that might be useful for debugging the classes in calls of the `bluemira_debug` method.